mod-audit-server/src/main/java/org/folio/builder/service/LogRecordBuilder.java

```diff
@@ -155,6 +156,16 @@ public LogRecordBuilder(Map<String, String> okapiHeaders, Context vertxContext)
       });
   }
 
+  public CompletableFuture<JsonObject> fetchUserDetails(JsonObject payload) {
```

Comment:
This method looks pretty the same as fetchUserBarcode. Is it possible to refactor it somehow just to prevent duplication?